### PR TITLE
Add color icons to VOI list

### LINKED ISSUE #547: Add color indication to VOI list
VOI list does not have any color indicator, so uset does not know which color repreesents which VOI on the image

### PR DESCRIPTION
**Pull request by rozchlastywacz**
Closes #547 
- #547 

image
image

edit: changed size of icons to match checkbox size:
image

### CODE REVIEW

**Review by Yoanka — APPROVED** [review]
Looks and works OK

File: pytripgui/view/qt_view_adapter.py [hunk 1]
```diff
@@ -162,11 +162,18 @@ def __init__(self, list_widget, checkable=False):
...
-    def fill(self, items, get_name: Callable):
+    def fill(self, items, get_name: Callable, get_color: Callable = None):
         self._ui.clear()
         self._items.clear()
         for item in items:
-            q_item = QListWidgetItem(get_name(item))
+            name = get_name(item)
+            q_item = QListWidgetItem(name)
+            if get_color is not None:
+                import numpy as np
```
Comment (grzanka) [inline pytripgui/view/qt_view_adapter.py:172]: why local import ?

File: pytripgui/view/qt_view_adapter.py [hunk 2]
```diff
@@ -162,11 +162,18 @@ def __init__(self, list_widget, checkable=False):
...
 
-    def fill(self, items, get_name: Callable):
+    def fill(self, items, get_name: Callable, get_color: Callable = None):
         self._ui.clear()
         self._items.clear()
         for item in items:
-            q_item = QListWidgetItem(get_name(item))
+            name = get_name(item)
+            q_item = QListWidgetItem(name)
+            if get_color is not None:
```
Comment (grzanka) [inline pytripgui/view/qt_view_adapter.py:171]:
```suggestion
            if get_color:
```

File: pytripgui/view/qt_view_adapter.py (hunk 1 above)
Comment (rozchlastywacz) [reply pytripgui/view/qt_view_adapter.py]: dunno

**Review by michal367 — APPROVED** [review]

**Pull request merged**